src/hooks/__internal__/useIsStickyFooterForm/useIsStickyFooterForm.ts

```diff
@@ -1,18 +0,0 @@
-import React, { useMemo } from "react";
-import Form from "../../../components/form/form.component";
-
-export default (children?: React.ReactNode) => {
-  const isStickyFooterForm = useMemo(
-    () =>
-      React.Children.toArray(children).some(
-        (child) =>
-          React.isValidElement(child) &&
-          (child.type as React.FunctionComponent)?.displayName ===
-            Form.displayName &&
-          child.props.stickyFooter
-      ),
-    [children]
-  );
```

Comment:
As this hook iterates over `children`, only `Form`s that are passed as direct React nodes will be checked. This is why CSS styling has been used instead :)